package.json

```diff
@@ -9,10 +9,11 @@
 	"bugs": {
 		"url": "https://github.com/spicetify/spicetify-marketplace/issues"
 	},
+	"packageManager": "pnpm@8.10.2",
 	"scripts": {
 		"build": "spicetify-creator",
 		"build:local": "spicetify-creator --out=dist --minify",
-		"build:prod": "pnpm build:local && pnpm copy:docs",
+		"build:prod": "npm run build:local && npm run copy:docs",
```

Comment:
I'm confused. Why are we changing this to npm?

Reply:
> why have pnpm here if we don't force to use it?

Reply:
Are we not forcing to use it? I don't get it

Reply:
> Are we not forcing to use it?

yes, we don't force to use pnpm